src/Files.App.Controls/ThemedIcon/ThemedIcon.Properties.cs

```diff
@@ -32,6 +32,7 @@ protected virtual void OnOutlineIconPropertyChanged(string oldValue, string newV
 		protected virtual void OnColorPropertyChanged(Brush oldValue, Brush newValue)
 		{
 			OnIconTypeChanged();
+			UpdateColor(newValue);
```

Comment:
I feel like this should probably be 
```suggestion
			OnIconColorChanged(newValue);
```